.github/conda_skip.yml

```diff
@@ -182,3 +182,5 @@ exclude:
     path: modules/nf-core/xeniumranger/resegment
   - profile: conda
     path: modules/nf-core/xeniumranger/import-segmentation
+  - profile: conda
+    path: modules/nf-core/parabricks/applybqsr
```

Comment:
that does nothing. this file should actually be deleted. Hmm, looks like the actual exclusion step is stuck in some other PR...

Reply:
ok then I will remove this again? or can I just leave it in?